app/game/game.js

```diff
@@ -1,13 +1,27 @@
-const { ROLES } = require("../constants");
+const { ROLES, GAME_TICK_DELAY_MS } = require("../constants");
+const { Threat, THREAT_COOLDOWN_SECONDS } = require("./threat");
+const { randomInt } = require("../utils.js");
+
 const GAMES = {
     
 };
 
+const selectThreatRoom = (avilableRooms, roomsWithThreats) => {
+    const rooms = avilableRooms.filter((room) => !(room in roomsWithThreats));
+    return rooms[randomInt(0, rooms.length - 1)];
+}
+
 const GAME = (humanUsername, aiUsername) => {
-    let gameLength = 4*60*1000;
+    let gameTime = 4*60;
     let HUMAN_USERNAME = humanUsername;
     let AI_USERNAME = aiUsername;
     let room = "0-0";
+    let threatCooldown = THREAT_COOLDOWN_SECONDS;
+
+    // Temp for testing: Spawns threat right away, TO DO: Determine when to spawn threats
+    const THREATS = [];
+    const AVAILABLE_ROOMS = ["0-0", "0-1", "0-2", "1-0", "1-1", "1-2", "2-0", "2-1", "2-2"]; // Richard: Yes I know it's hardcoded, we can make a dynamic room generator later TO DO 
```

Comment:
Actually, my main point of objection is their representation as strings rather than a list like [row, col] ;)

Reply:
Yeah it's hardcoded for now, and I didn't make AVAILABLE_ROWS a 2d array because the code would be a tiny bit more complicated. This way we can just use indexOf and see if a room is in the array